google/ggrc-core
# GGRC-4408 Auditor/Audit captain is duplicated after deleting other auditors

# Issue description

When we delete some role this action can delete another acl role on the backend.
In this case we get in response less objects then was in request.
But canJs is merging array-attributes not replacing.
As result it doesn't remove redundant element.

# Steps to test the changes

Steps to reproduce:
1. Open Audit summary page
2. Add at least 3 auditors or Audit captains > Save
3. Remove 2 auditors/audit captains > Save

*Actual Result:* Last auditor/audit captain is duplicated
*Expected Result:* Last auditor/audit captain should not be duplicated

# Solution description

Exclude acl-roles without id after getting response.

# Sanity checklist




# PR Review checklist
